tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -352,13 +353,27 @@ def test_monitoring_after_rebooting_master_node(self, nodes, pods):
 
         # Get the master node list
         master_nodes = get_typed_nodes(node_type='master')
-
-        # Reboot one after one master nodes
-        for node in master_nodes:
-            nodes.restart_nodes([node])
-
-            wait_for_nodes_status_and_prometheus_health_check(pods)
-
+        # Check if LSO is present
+        local_storage_operator = ocp.OCP(
+            kind=constants.POD, namespace='local-storage'
+        )
+        if local_storage_operator.wait_for_resource(
+            condition=constants.STATUS_RUNNING,
+            selector=constants.LOCAL_STORAGE_OPERATOR_LABEL,
+            timeout=600
+        ):
+            # Enter in the node and execute reboot
+            for node in master_nodes:
+                utils.run_cmd(f"oc debug node/{node}")
+                utils.run_cmd("reboot")
+
+                wait_for_nodes_status_and_prometheus_health_check(pods)
+        else:
+            # Reboot one after one master nodes
+            for node in master_nodes:
+                nodes.restart_nodes(nodes=[node])
+
+                wait_for_nodes_status_and_prometheus_health_check(pods)
```

Comment:
I suggest we wait for https://github.com/red-hat-storage/ocs-ci/pull/2006 to be merged, which will allow us running restart using IPMI, and this change will not be needed as nodes.restart_nodes() will accommodate this

Reply:
ack!